src/pretix/control/templates/pretixcontrol/event/plugins.html

```diff
@@ -10,20 +10,41 @@ <h1>{% trans "Available plugins" %}</h1>
             software functionality, connect your event to third-party services, or apply other forms of customizations.
         {% endblocktrans %}
     </p>
+    {% if "success" in request.GET %}
+        <div class="alert alert-success">
+            {% trans "Your changes have been saved." %}
+        </div>
+    {% endif %}
+    <p>
+        <div class="row">
+            <div class="col-lg-10">
+                <input type="search" id="plugin_search_input" class="form-control" placeholder="{% trans "Search" %}" autofocus>
```

Comment:
No autofocus please, see also #4874

```suggestion
                <input type="search" id="plugin_search_input" class="form-control" placeholder="{% trans "Search" %}">
```